internal/storage/sql.go

```diff
@@ -90,7 +90,7 @@ var (
 	SetHosts = `UPDATE hosts SET data = ?, lastmodified_time = datetime('now','localtime') WHERE id = ?`
 
 	// select hosts
-	SelectHosts = `SELECT * FROM hosts`
+	SelectHosts = `SELECT id,data,lastmodified_time FROM hosts`
```

Comment:
Lack of space, also below, it's better to check it.

Reply:
I see,I will fix it.